tox.ini

```diff
@@ -45,7 +61,7 @@ deps =
     oldestdeps: sunpy<4.0
     oldestdeps: astropy<4.3
     oldestdeps: gwcs<0.16
-    oldestdeps: numpy<1.18
+    oldestdeps: numpy<1.23
     oldestdeps: matplotlib<3.3
     oldestdeps: mpl_animators<1.1
     oldestdeps: reproject==0.7.1
```

Comment:
We should be able to use minimum_dependencies here as we don't have the recursive deps like sunpy does.